2024-09-08T17:15:18-07:00 SynologyNAS syno_swap_ctl[7452]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 1 drive (out of 4). "
2024-09-09T08:24:46+08:00 Synology_NAS syno_swap_ctl[27611]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md1" Error="mdadm: stopped /dev/md1 "
2024-09-10T07:07:21+08:00 Synology_NAS syno_swap_ctl[6844]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 1 drive (out of 4). "
2024-09-11T20:04:05+08:00 Synology_NAS syno_swap_ctl[6898]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata3p2" Error="mdadm: /dev/md1 has been started with 1 drive (out of 4). "
2024-09-11T20:08:59+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/mdadm --grow /dev/md0 --force -n 12" Error=""
2024-09-11T20:09:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/mdadm --grow /dev/md1 --force -n 12" Error=""
2024-09-11T20:09:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-11T20:09:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14454]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-11T20:09:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14454]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z27323318272 -tfd -F /dev/sata3" Error=""
2024-09-11T20:09:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p3" Error="mdadm: Unrecognised md component device - /dev/sata3p3 "
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md2 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata3p3" Error="mdadm: array /dev/md2 started. "
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md2 bs=512 count=1 status=none" Error=""
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md2" Error=""
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg1 /dev/md2" Error=""
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg1 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-11T20:09:04+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg1 -n volume_1 --size 13340672M" Error=""
2024-09-11T20:09:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg1/volume_1" Error=""
2024-09-11T20:09:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_0 --nodiscard --force --label 2024.09.11-12:09:05 v69057 --metadata dup" Error=""
2024-09-11T20:09:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_0 /tmp/mount.7hRZv9" Error=""
2024-09-11T20:09:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.7hRZv9/@syno" Error=""
2024-09-11T20:09:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.7hRZv9" Error=""
2024-09-11T20:09:06+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume1" Error=""
2024-09-11T20:09:11+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create[14427]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume1" Error=""
2024-09-11T20:09:48+08:00 Synology_NAS cloud-control[20747]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/@synologydrive"
2024-09-11T20:10:08+08:00 Synology_NAS synodrivesettings[22025]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/@SynoDrive/office"
2024-09-11T20:10:34+08:00 Synology_NAS synowebapi_SYNO.Core.User.Home_1_set[23538]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/homes"
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete -1 /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk -N1 -uS -q -f -j8192 -z16777216 -tfd -F /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk -N2 -uS -q -f -j0 -z4194304 -tfd -F /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p1" Error="mdadm: Unrecognised md component device - /dev/sata2p1 "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p2" Error="mdadm: Unrecognised md component device - /dev/sata2p2 "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata2p1" Error="mdadm: added /dev/sata2p1 "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata2p2" Error="mdadm: added /dev/sata2p2 "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[28059]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[28059]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z7792590496 -tfd -F /dev/sata2" Error=""
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p3" Error="mdadm: Unrecognised md component device - /dev/sata2p3 "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md3 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata2p3" Error="mdadm: array /dev/md3 started. "
2024-09-11T20:19:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md3 bs=512 count=1 status=none" Error=""
2024-09-11T20:19:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md3" Error=""
2024-09-11T20:19:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg2 /dev/md3" Error=""
2024-09-11T20:19:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[27921]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg2 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-11T20:20:58+08:00 Synology_NAS cloud-control[30672]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume1/@tmp/cloud.tmp.dir/X9g4Gt9W"
2024-09-11T20:20:58+08:00 Synology_NAS cloud-drive[30679]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume1/@SynoDrive/office"
2024-09-11T20:28:51+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_delete[3806]: space_error_log.c:47 command="/bin/umount -R -A -k -f /volume1" Error=""
2024-09-11T20:28:51+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_delete[3806]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-11T20:28:51+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_delete[3806]: space_error_log.c:47 command="/sbin/lvremove -f /dev/vg1/volume_1" Error=""
2024-09-11T20:29:52+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg1 -n volume_1 --size 13340672M" Error="WARNING: btrfs signature detected on /dev/vg1/volume_1 at offset 65600. Wipe it? [y/n]: n   Aborted wiping of btrfs.   1 existing signature left on the device. "
2024-09-11T20:29:53+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg1/volume_1" Error=""
2024-09-11T20:29:53+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_0 --nodiscard --force --label 2024.09.11-12:29:53 v69057 --metadata dup" Error=""
2024-09-11T20:29:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_0 /tmp/mount.sf4XJ6" Error=""
2024-09-11T20:29:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.sf4XJ6/@syno" Error=""
2024-09-11T20:29:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.sf4XJ6" Error=""
2024-09-11T20:29:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume1" Error=""
2024-09-11T20:29:58+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[7118]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume1" Error=""
2024-09-11T20:30:33+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg2 -n volume_2 --size 3804160M" Error="WARNING: atari signature detected on /dev/vg2/volume_2 at offset 478. Wipe it? [y/n]: n   Aborted wiping of atari.   1 existing signature left on the device. "
2024-09-11T20:30:33+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_1 none /dev/vg2/volume_2" Error=""
2024-09-11T20:30:34+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_1 --nodiscard --force --label 2024.09.11-12:30:34 v69057 --metadata dup" Error=""
2024-09-11T20:30:34+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_1 /tmp/mount.QNgxfd" Error=""
2024-09-11T20:30:34+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.QNgxfd/@syno" Error=""
2024-09-11T20:30:34+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.QNgxfd" Error=""
2024-09-11T20:30:35+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume2" Error=""
2024-09-11T20:30:38+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[8494]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume2" Error=""
2024-09-11T20:53:20+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[12587]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume2/MG04ACA400N"
2024-09-12T00:26:26+08:00 Synology_NAS spacetool[19501]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool[19502]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS synovspace[19582]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS synovspace[19582]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19627]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19626]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19626]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19627]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19627]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-12T00:26:27+08:00 Synology_NAS spacetool.shared[19626]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-12T00:28:51+08:00 Synology_NAS syno_swap_ctl[7210]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata4p2 /dev/sata3p2" Error="mdadm: /dev/md1 has been started with 2 drives (out of 12). "
2024-09-12T00:30:22+08:00 Synology_NAS spacetool.shared[9806]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata3p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-12T00:30:22+08:00 Synology_NAS spacetool.shared[9807]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata4p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-12T00:30:22+08:00 Synology_NAS spacetool.shared[9722]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-12T00:30:22+08:00 Synology_NAS spacetool.shared[9722]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-12T00:30:24+08:00 Synology_NAS synovspace[10988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_0 none /dev/vg2/volume_2" Error=""
2024-09-12T00:30:24+08:00 Synology_NAS synovspace[10988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_1 none /dev/vg1/volume_1" Error=""
2024-09-12T00:30:25+08:00 Synology_NAS spacetool[11414]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume1" Error=""
2024-09-12T00:30:25+08:00 Synology_NAS spacetool[11408]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume2" Error=""
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete -1 /dev/sata2" Error="Warning: The format of the GPT partition table is version 20000, which is newer than what Parted can recognise.  Please report this! "
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata2" Error="Warning: The format of the GPT partition table is version 20000, which is newer than what Parted can recognise.  Please report this! "
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata2" Error=""
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk -N1 -uS -q -f -j8192 -z16777216 -tfd -F /dev/sata2" Error=""
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk -N2 -uS -q -f -j0 -z4194304 -tfd -F /dev/sata2" Error=""
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p1" Error="mdadm: Unrecognised md component device - /dev/sata2p1 "
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p2" Error="mdadm: Unrecognised md component device - /dev/sata2p2 "
2024-09-12T00:31:54+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata2p1" Error="mdadm: added /dev/sata2p1 "
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata2p2" Error="mdadm: added /dev/sata2p2 "
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata2" Error=""
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[16049]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata2" Error=""
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[16049]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z1932078496 -tfd -F /dev/sata2" Error=""
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata2p3" Error="mdadm: Unrecognised md component device - /dev/sata2p3 "
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md4 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata2p3" Error="mdadm: array /dev/md4 started. "
2024-09-12T00:31:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md4 bs=512 count=1 status=none" Error=""
2024-09-12T00:31:56+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md4" Error=""
2024-09-12T00:31:56+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg3 /dev/md4" Error=""
2024-09-12T00:31:56+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[15882]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg3 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-12T00:32:14+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg3 -n volume_3 --size 943104M" Error=""
2024-09-12T00:32:15+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_2 none /dev/vg3/volume_3" Error=""
2024-09-12T00:32:16+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_2 --nodiscard --force --label 2024.09.11-16:32:16 v69057 --metadata dup" Error=""
2024-09-12T00:32:16+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_2 /tmp/mount.S2GBVR" Error=""
2024-09-12T00:32:16+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.S2GBVR/@syno" Error=""
2024-09-12T00:32:16+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.S2GBVR" Error=""
2024-09-12T00:32:17+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume3" Error=""
2024-09-12T00:32:21+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[16584]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume3" Error=""
2024-09-12T00:43:02+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[20105]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/MG08ACA14TE"
2024-09-12T00:44:22+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[21179]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume3/ST1000LM0335"
2024-09-12T10:40:23+08:00 Synology_NAS synoshare[10784]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/docker"
2024-09-13T08:55:43+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[23565]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume2" Error=""
2024-09-13T08:55:55+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[23699]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume3" Error=""
2024-09-13T08:56:02+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[23777]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume1" Error=""
2024-09-13T09:02:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[26066]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume2" Error=""
2024-09-13T09:02:14+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[26139]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume3" Error=""
2024-09-13T09:02:21+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[26222]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume1" Error=""
2024-09-13T09:03:52+08:00 Synology_NAS hotplugd[12670]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-13T09:03:58+08:00 Synology_NAS hotplugd[12670]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error="The disk contains an unclean file system (0, 0). The file system wasn't safely closed on Windows. Fixing. "
2024-09-14T22:43:08+08:00 Synology_NAS spacetool[11182]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-14T22:43:08+08:00 Synology_NAS spacetool[11183]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-14T22:43:09+08:00 Synology_NAS spacetool[11181]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS synovspace[11703]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS synovspace[11703]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS synovspace[11703]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11765]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11768]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11766]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11765]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11766]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-14T22:43:10+08:00 Synology_NAS spacetool.shared[11768]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-14T22:43:11+08:00 Synology_NAS spacetool.shared[11768]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-14T22:43:11+08:00 Synology_NAS spacetool.shared[11766]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-14T22:43:11+08:00 Synology_NAS spacetool.shared[11765]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-14T23:15:16+08:00 Synology_NAS syno_swap_ctl[7158]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 3 drives (out of 12). "
2024-09-14T23:16:48+08:00 Synology_NAS spacetool.shared[11402]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-14T23:16:48+08:00 Synology_NAS spacetool.shared[11404]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-14T23:16:48+08:00 Synology_NAS spacetool.shared[11401]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-14T23:16:49+08:00 Synology_NAS spacetool.shared[11148]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-14T23:16:49+08:00 Synology_NAS spacetool.shared[11148]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-14T23:16:49+08:00 Synology_NAS spacetool.shared[11148]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-14T23:16:50+08:00 Synology_NAS synovspace[11793]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_0 none /dev/vg3/volume_3" Error=""
2024-09-14T23:16:50+08:00 Synology_NAS synovspace[11793]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_1 none /dev/vg2/volume_2" Error=""
2024-09-14T23:16:50+08:00 Synology_NAS synovspace[11793]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_2 none /dev/vg1/volume_1" Error=""
2024-09-14T23:16:51+08:00 Synology_NAS spacetool[12269]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume3" Error=""
2024-09-14T23:16:51+08:00 Synology_NAS spacetool[12266]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume1" Error=""
2024-09-14T23:16:52+08:00 Synology_NAS spacetool[12257]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume2" Error=""
2024-09-14T23:16:58+08:00 Synology_NAS hotplugd[14066]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-14T23:17:05+08:00 Synology_NAS hotplugd[14066]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete -1 /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk -N1 -uS -q -f -j8192 -z16777216 -tfd -F /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk -N2 -uS -q -f -j0 -z4194304 -tfd -F /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p1" Error="mdadm: Unrecognised md component device - /dev/sata3p1 "
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p2" Error="mdadm: Unrecognised md component device - /dev/sata3p2 "
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata3p1" Error="mdadm: added /dev/sata3p1 "
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata3p2" Error="mdadm: added /dev/sata3p2 "
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6838]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6838]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z27323318272 -tfd -F /dev/sata3" Error=""
2024-09-15T00:00:22+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p3" Error="mdadm: Unrecognised md component device - /dev/sata3p3 "
2024-09-15T00:00:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md5 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata3p3" Error="mdadm: array /dev/md5 started. "
2024-09-15T00:00:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md5 bs=512 count=1 status=none" Error=""
2024-09-15T00:00:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md5" Error=""
2024-09-15T00:00:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg4 /dev/md5" Error=""
2024-09-15T00:00:23+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[6648]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg4 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-15T00:01:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_delete[7968]: space_error_log.c:47 command="/sbin/vgremove -f /dev/vg4" Error=""
2024-09-15T00:01:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_delete[7968]: space_error_log.c:47 command="/sbin/pvremove -ff -y /dev/md5" Error=""
2024-09-15T00:01:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_delete[7968]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-15T00:01:03+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_delete[8068]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p3" Error=""
2024-09-15T00:03:46+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-15T00:03:46+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9641]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata3" Error=""
2024-09-15T00:03:46+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9641]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z27323318272 -tfd -F /dev/sata3" Error=""
2024-09-15T00:03:46+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata3p3" Error="mdadm: Unrecognised md component device - /dev/sata3p3 "
2024-09-15T00:03:46+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md5 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata3p3" Error="mdadm: array /dev/md5 started. "
2024-09-15T00:03:47+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md5 bs=512 count=1 status=none" Error=""
2024-09-15T00:03:47+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md5" Error=""
2024-09-15T00:03:47+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg4 /dev/md5" Error=""
2024-09-15T00:03:47+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[9628]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg4 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-15T00:04:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg4 -n volume_4 --size 13340672M" Error=""
2024-09-15T00:04:06+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg4/volume_4" Error=""
2024-09-15T00:04:06+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_3 --nodiscard --force --label 2024.09.14-16:04:06 v69057 --metadata dup" Error=""
2024-09-15T00:04:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_3 /tmp/mount.7wtRoD" Error=""
2024-09-15T00:04:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.7wtRoD/@syno" Error=""
2024-09-15T00:04:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.7wtRoD" Error=""
2024-09-15T00:04:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume4" Error=""
2024-09-15T00:04:11+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[10247]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume4" Error=""
2024-09-15T00:05:19+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[11890]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/WDC WUH721414ALE604"
2024-09-15T00:50:06+08:00 Synology_NAS spacetool[27409]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-15T00:50:06+08:00 Synology_NAS spacetool[27411]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-15T00:50:06+08:00 Synology_NAS spacetool[27410]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-15T00:50:08+08:00 Synology_NAS spacetool[27413]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS synovspace[27493]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS synovspace[27493]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS synovspace[27493]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS synovspace[27493]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27574]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27573]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27576]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27578]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27574]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27576]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27578]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27573]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27578]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27573]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27576]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-15T00:50:09+08:00 Synology_NAS spacetool.shared[27574]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-15T01:12:42+08:00 Synology_NAS syno_swap_ctl[7244]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[11057]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[11054]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[11058]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[11053]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[10736]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[10736]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[10736]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-15T01:14:14+08:00 Synology_NAS spacetool.shared[10736]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS synovspace[11519]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS synovspace[11519]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS synovspace[11519]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS synovspace[11519]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-15T01:14:17+08:00 Synology_NAS spacetool[11926]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-15T01:14:17+08:00 Synology_NAS spacetool[11931]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS spacetool[11961]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-15T01:14:16+08:00 Synology_NAS spacetool[11944]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-15T01:14:21+08:00 Synology_NAS hotplugd[13909]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-15T01:14:26+08:00 Synology_NAS hotplugd[13909]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-15T01:17:47+08:00 Synology_NAS syno_swap_ctl[7256]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-15T01:20:29+08:00 Synology_NAS syno_swap_ctl[7181]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-15T01:22:00+08:00 Synology_NAS spacetool.shared[10722]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10724]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10726]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10721]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10274]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10274]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10274]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-15T01:22:01+08:00 Synology_NAS spacetool.shared[10274]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS synovspace[11560]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS synovspace[11560]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS synovspace[11560]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS synovspace[11560]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS spacetool[11941]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS spacetool[11982]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS spacetool[12012]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-15T01:22:03+08:00 Synology_NAS spacetool[12079]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-15T01:22:11+08:00 Synology_NAS hotplugd[14367]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-15T01:22:13+08:00 Synology_NAS synoraidtool[16431]: fs_btrfs_cmd.c:222 command="/sbin/btrfs scrub start -B -c3 -n7 /volume1"
2024-09-15T01:22:17+08:00 Synology_NAS hotplugd[14367]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-15T01:29:23+08:00 Synology_NAS syno_swap_ctl[7244]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11327]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11329]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11331]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11326]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11265]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11265]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11265]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-15T01:30:55+08:00 Synology_NAS spacetool.shared[11265]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS synovspace[11791]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS synovspace[11791]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS synovspace[11791]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS synovspace[11791]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS spacetool[12173]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS spacetool[12192]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-15T01:30:57+08:00 Synology_NAS spacetool[12221]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-15T01:30:58+08:00 Synology_NAS spacetool[12217]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-15T01:31:04+08:00 Synology_NAS hotplugd[14513]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-15T01:31:06+08:00 Synology_NAS synoraidtool[15476]: fs_btrfs_cmd.c:222 command="/sbin/btrfs scrub start -B -c3 -n7 /volume1"
2024-09-15T01:31:12+08:00 Synology_NAS hotplugd[14513]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-15T05:04:34+08:00 Synology_NAS syno_swap_ctl[6849]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 1 drive (out of 12). "
2024-09-15T05:06:06+08:00 Synology_NAS spacetool.shared[9256]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-15T05:06:07+08:00 Synology_NAS spacetool.shared[9213]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-15T05:06:08+08:00 Synology_NAS synovspace[9383]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_0 none /dev/vg3/volume_3" Error=""
2024-09-15T05:06:10+08:00 Synology_NAS spacetool[10224]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume3" Error=""
2024-09-15T05:13:35+08:00 Synology_NAS syno_swap_ctl[7118]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 1 drive (out of 12). "
2024-09-15T05:15:09+08:00 Synology_NAS spacetool.shared[10854]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-15T05:15:09+08:00 Synology_NAS spacetool.shared[10858]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-15T05:15:09+08:00 Synology_NAS spacetool.shared[10856]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-15T05:15:10+08:00 Synology_NAS spacetool.shared[10853]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-15T05:15:11+08:00 Synology_NAS spacetool.shared[10570]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-15T05:15:11+08:00 Synology_NAS spacetool.shared[10570]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-15T05:15:11+08:00 Synology_NAS spacetool.shared[10570]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-15T05:15:11+08:00 Synology_NAS spacetool.shared[10570]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-15T05:15:13+08:00 Synology_NAS synovspace[11481]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-15T05:15:13+08:00 Synology_NAS synovspace[11481]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-15T05:15:13+08:00 Synology_NAS synovspace[11481]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-15T05:15:13+08:00 Synology_NAS synovspace[11481]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-15T05:15:15+08:00 Synology_NAS spacetool[11866]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-15T05:15:15+08:00 Synology_NAS spacetool[11916]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-15T05:15:15+08:00 Synology_NAS spacetool[11897]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-15T05:15:15+08:00 Synology_NAS spacetool[11879]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-15T05:15:24+08:00 Synology_NAS hotplugd[14585]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-15T05:15:29+08:00 Synology_NAS synoraidtool[16893]: fs_btrfs_cmd.c:222 command="/sbin/btrfs scrub start -B -c3 -n7 /volume3"
2024-09-15T05:15:31+08:00 Synology_NAS hotplugd[14585]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-15T05:34:57+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm --action=frozen /dev/md0" Error=""
2024-09-15T05:34:57+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm --action=frozen /dev/md1" Error=""
2024-09-15T05:34:57+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata3p1" Error="mdadm: added /dev/sata3p1 "
2024-09-15T05:34:59+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata3p2" Error="mdadm: added /dev/sata3p2 "
2024-09-15T05:35:00+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata4p1" Error="mdadm: added /dev/sata4p1 "
2024-09-15T05:35:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata4p2" Error="mdadm: added /dev/sata4p2 "
2024-09-15T05:35:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata5p1" Error="mdadm: added /dev/sata5p1 "
2024-09-15T05:35:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata5p2" Error="mdadm: added /dev/sata5p2 "
2024-09-15T05:35:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm --action=idle /dev/md0" Error=""
2024-09-15T05:35:07+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Storage_1_repair_system_partition[27201]: space_error_log.c:47 command="/sbin/mdadm --action=idle /dev/md1" Error=""
2024-09-16T05:35:28+08:00 Synology_NAS spacetool[11712]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-16T05:35:28+08:00 Synology_NAS spacetool[11715]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-16T05:35:29+08:00 Synology_NAS spacetool[11713]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-16T05:35:30+08:00 Synology_NAS spacetool[11711]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS synovspace[12240]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS synovspace[12240]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS synovspace[12240]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS synovspace[12240]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12320]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12321]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12323]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12325]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12320]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12325]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12323]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12321]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12323]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12325]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-16T05:35:31+08:00 Synology_NAS spacetool.shared[12320]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-16T05:35:32+08:00 Synology_NAS spacetool.shared[12321]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-16T05:38:04+08:00 Synology_NAS syno_swap_ctl[7172]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[11087]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[11089]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[11086]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[11091]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[10932]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[10932]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[10932]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-16T05:39:36+08:00 Synology_NAS spacetool.shared[10932]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-16T05:39:38+08:00 Synology_NAS synovspace[11533]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-16T05:39:38+08:00 Synology_NAS synovspace[11533]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-16T05:39:38+08:00 Synology_NAS synovspace[11533]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-16T05:39:38+08:00 Synology_NAS synovspace[11533]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-16T05:39:39+08:00 Synology_NAS spacetool[11938]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-16T05:39:39+08:00 Synology_NAS spacetool[12005]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-16T05:39:39+08:00 Synology_NAS spacetool[11928]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-16T05:39:39+08:00 Synology_NAS spacetool[11968]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-16T05:39:47+08:00 Synology_NAS hotplugd[14199]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-16T05:39:53+08:00 Synology_NAS hotplugd[14199]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-16T19:28:50+08:00 Synology_NAS spacetool[18679]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-16T19:28:51+08:00 Synology_NAS spacetool[18676]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-16T19:28:51+08:00 Synology_NAS spacetool[18677]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-16T19:28:52+08:00 Synology_NAS spacetool[18675]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS synovspace[19204]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS synovspace[19204]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS synovspace[19204]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS synovspace[19204]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19285]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19284]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19289]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19287]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19285]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19284]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19289]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19287]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19289]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19284]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19287]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-16T19:28:53+08:00 Synology_NAS spacetool.shared[19285]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-16T19:31:26+08:00 Synology_NAS syno_swap_ctl[7202]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-16T19:32:57+08:00 Synology_NAS spacetool.shared[10033]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-16T19:32:57+08:00 Synology_NAS spacetool.shared[10035]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-16T19:32:57+08:00 Synology_NAS spacetool.shared[10037]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-16T19:32:57+08:00 Synology_NAS spacetool.shared[10032]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-16T19:32:58+08:00 Synology_NAS spacetool.shared[9972]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-16T19:32:58+08:00 Synology_NAS spacetool.shared[9972]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-16T19:32:58+08:00 Synology_NAS spacetool.shared[9972]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-16T19:32:58+08:00 Synology_NAS spacetool.shared[9972]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS synovspace[11515]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS synovspace[11515]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS synovspace[11515]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS synovspace[11515]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS spacetool[11939]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS spacetool[11943]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-16T19:33:00+08:00 Synology_NAS spacetool[11917]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-16T19:33:01+08:00 Synology_NAS spacetool[12029]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-16T19:33:08+08:00 Synology_NAS hotplugd[14152]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-16T19:33:14+08:00 Synology_NAS hotplugd[14152]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-17T04:54:32+08:00 Synology_NAS spacetool[21436]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-17T04:57:05+08:00 Synology_NAS syno_swap_ctl[7189]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-17T04:58:36+08:00 Synology_NAS spacetool.shared[10052]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-17T04:58:36+08:00 Synology_NAS spacetool.shared[10051]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-17T04:58:36+08:00 Synology_NAS spacetool.shared[10053]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-17T04:58:36+08:00 Synology_NAS spacetool.shared[10055]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-17T04:58:37+08:00 Synology_NAS spacetool.shared[9978]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-17T04:58:37+08:00 Synology_NAS spacetool.shared[9978]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-17T04:58:37+08:00 Synology_NAS spacetool.shared[9978]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-17T04:58:37+08:00 Synology_NAS spacetool.shared[9978]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-17T04:58:39+08:00 Synology_NAS synovspace[11582]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-17T04:58:39+08:00 Synology_NAS synovspace[11582]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-17T04:58:39+08:00 Synology_NAS synovspace[11582]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-17T04:58:39+08:00 Synology_NAS synovspace[11582]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-17T04:58:41+08:00 Synology_NAS spacetool[12018]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-17T04:58:41+08:00 Synology_NAS spacetool[12012]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-17T04:58:41+08:00 Synology_NAS spacetool[11969]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-17T04:58:41+08:00 Synology_NAS spacetool[12008]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-17T04:58:48+08:00 Synology_NAS hotplugd[14169]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-17T04:58:55+08:00 Synology_NAS hotplugd[14169]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-17T05:46:40+08:00 Synology_NAS spacetool[2902]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-17T05:46:40+08:00 Synology_NAS spacetool[2899]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-17T05:46:41+08:00 Synology_NAS spacetool[2900]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-17T05:46:42+08:00 Synology_NAS spacetool[2898]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS synovspace[3428]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS synovspace[3428]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS synovspace[3428]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS synovspace[3428]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3511]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3515]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3510]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3513]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3511]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3515]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3510]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3513]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3515]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3510]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3513]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-17T05:46:43+08:00 Synology_NAS spacetool.shared[3511]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-17T05:49:15+08:00 Synology_NAS syno_swap_ctl[7235]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[10040]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[10039]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[10042]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[10044]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[9971]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-17T05:50:46+08:00 Synology_NAS spacetool.shared[9971]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-17T05:50:47+08:00 Synology_NAS spacetool.shared[9971]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-17T05:50:47+08:00 Synology_NAS spacetool.shared[9971]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-17T05:50:49+08:00 Synology_NAS synovspace[11523]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-17T05:50:49+08:00 Synology_NAS synovspace[11523]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-17T05:50:49+08:00 Synology_NAS synovspace[11523]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-17T05:50:49+08:00 Synology_NAS synovspace[11523]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-17T05:50:50+08:00 Synology_NAS spacetool[11943]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-17T05:50:50+08:00 Synology_NAS spacetool[11973]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-17T05:50:50+08:00 Synology_NAS spacetool[11924]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-17T05:50:50+08:00 Synology_NAS spacetool[11958]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-17T05:50:58+08:00 Synology_NAS hotplugd[14193]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-17T05:51:03+08:00 Synology_NAS hotplugd[14193]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-17T10:41:56+08:00 Synology_NAS spacetool[4906]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-17T10:41:56+08:00 Synology_NAS spacetool[4903]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-17T10:41:56+08:00 Synology_NAS spacetool[4904]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-17T10:41:58+08:00 Synology_NAS spacetool[4902]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS synovspace[5433]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS synovspace[5433]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS synovspace[5433]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS synovspace[5433]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5514]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5515]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5517]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5528]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5514]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5528]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5517]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5515]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5528]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5514]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5517]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-17T10:41:59+08:00 Synology_NAS spacetool.shared[5515]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-17T10:44:30+08:00 Synology_NAS syno_swap_ctl[7235]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10872]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10875]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata5p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10873]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata4p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10870]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10246]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10246]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10246]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-17T10:46:02+08:00 Synology_NAS spacetool.shared[10246]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS synovspace[11521]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS synovspace[11521]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS synovspace[11521]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS synovspace[11521]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS spacetool[11957]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS spacetool[11925]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-17T10:46:04+08:00 Synology_NAS spacetool[11959]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-17T10:46:05+08:00 Synology_NAS spacetool[11961]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-17T10:46:12+08:00 Synology_NAS hotplugd[14211]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-17T10:46:17+08:00 Synology_NAS hotplugd[14211]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-20T07:36:44+08:00 Synology_NAS spacetool[9434]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-20T07:36:44+08:00 Synology_NAS spacetool[9431]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-20T07:36:45+08:00 Synology_NAS spacetool[9432]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-20T07:36:47+08:00 Synology_NAS spacetool[9430]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-20T07:36:48+08:00 Synology_NAS synovspace[10400]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-20T07:36:48+08:00 Synology_NAS synovspace[10400]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-20T07:36:48+08:00 Synology_NAS synovspace[10400]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-20T07:36:48+08:00 Synology_NAS synovspace[10400]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10481]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10485]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10480]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10483]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10480]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10481]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10485]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10483]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10483]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10485]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10480]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-20T07:36:49+08:00 Synology_NAS spacetool.shared[10481]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-20T07:49:47+08:00 Synology_NAS syno_swap_ctl[7332]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata6p2 /dev/sata5p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 4 drives (out of 12). "
2024-09-20T07:51:18+08:00 Synology_NAS spacetool.shared[10604]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-20T07:51:18+08:00 Synology_NAS spacetool.shared[10605]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-20T07:51:18+08:00 Synology_NAS spacetool.shared[10609]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata6p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-20T07:51:18+08:00 Synology_NAS spacetool.shared[10608]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata5p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-20T07:51:19+08:00 Synology_NAS spacetool.shared[10155]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-20T07:51:19+08:00 Synology_NAS spacetool.shared[10155]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-20T07:51:19+08:00 Synology_NAS spacetool.shared[10155]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-20T07:51:19+08:00 Synology_NAS spacetool.shared[10155]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS synovspace[11688]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_0 none /dev/vg4/volume_4" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS synovspace[11688]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_1 none /dev/vg3/volume_3" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS synovspace[11688]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_2 none /dev/vg2/volume_2" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS synovspace[11688]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_3 none /dev/vg1/volume_1" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS spacetool[12117]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume3" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS spacetool[12069]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume4" Error=""
2024-09-20T07:51:21+08:00 Synology_NAS spacetool[12107]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume2" Error=""
2024-09-20T07:51:22+08:00 Synology_NAS spacetool[12112]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume1" Error=""
2024-09-20T07:51:28+08:00 Synology_NAS hotplugd[13990]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-20T07:51:33+08:00 Synology_NAS hotplugd[13990]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete -1 /dev/sata4" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata4" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk -M1 /dev/sata4" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk -N1 -uS -q -f -j8192 -z16777216 -tfd -F /dev/sata4" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk -N2 -uS -q -f -j0 -z4194304 -tfd -F /dev/sata4" Error=""
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata4p1" Error="mdadm: Unrecognised md component device - /dev/sata4p1 "
2024-09-20T07:54:24+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata4p2" Error="mdadm: Unrecognised md component device - /dev/sata4p2 "
2024-09-20T07:54:25+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm /dev/md0 -a /dev/sata4p1" Error="mdadm: added /dev/sata4p1 "
2024-09-20T07:54:26+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -a /dev/sata4p2" Error="mdadm: added /dev/sata4p2 "
2024-09-20T07:54:26+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata4" Error=""
2024-09-20T07:54:26+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21525]: space_error_log.c:47 command="/sbin/sfdisk --fast-delete 3:128 /dev/sata4" Error=""
2024-09-20T07:54:26+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21525]: space_error_log.c:47 command="/sbin/sfdisk -N3  -uS -q -f -j262144 -z31230312448 -tfd -F /dev/sata4" Error=""
2024-09-20T07:54:26+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm --zero-superblock /dev/sata4p3" Error="mdadm: Unrecognised md component device - /dev/sata4p3 "
2024-09-20T07:54:27+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/mdadm -C /dev/md6 -e 1.2 -amd --assume-clean -R -l1 -f -n1 -x0 /dev/sata4p3" Error="mdadm: array /dev/md6 started. "
2024-09-20T07:54:27+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/bin/dd if=/dev/zero of=/dev/md6 bs=512 count=1 status=none" Error=""
2024-09-20T07:54:27+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/pvcreate -ff -y --metadatasize 512K /dev/md6" Error=""
2024-09-20T07:54:28+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/vgcreate --physicalextentsize 4m /dev/vg5 /dev/md6" Error=""
2024-09-20T07:54:28+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Pool_1_create[21215]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg5 -n syno_vg_reserved_area --size 12M" Error=""
2024-09-20T07:54:59+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/sbin/lvcreate /dev/vg5 -n volume_5 --size 15248384M" Error=""
2024-09-20T07:55:00+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 15614345216 -p dummy cachedev_4 none /dev/vg5/volume_5" Error=""
2024-09-20T07:55:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/sbin/mkfs.btrfs /dev/mapper/cachedev_4 --nodiscard --force --label 2024.09.19-23:55:01 v69057 --metadata dup" Error=""
2024-09-20T07:55:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/bin/mount /dev/mapper/cachedev_4 /tmp/mount.RYgqoG" Error=""
2024-09-20T07:55:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/sbin/btrfs subvolume create /tmp/mount.RYgqoG/@syno" Error=""
2024-09-20T07:55:01+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/sbin/btrfs subvolume set-default 256 /tmp/mount.RYgqoG" Error=""
2024-09-20T07:55:02+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_4 /volume5" Error=""
2024-09-20T07:55:09+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_create_on_existing_pool[22714]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume5" Error=""
2024-09-20T07:56:46+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[26065]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume5/MG08ACA16TE"
2024-09-20T11:29:05+08:00 Synology_NAS spacetool[1577]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume5" Error=""
2024-09-20T11:29:05+08:00 Synology_NAS spacetool[1571]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-09-20T11:29:05+08:00 Synology_NAS spacetool[1572]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-09-20T11:29:06+08:00 Synology_NAS spacetool[1573]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-09-20T11:29:08+08:00 Synology_NAS spacetool[1575]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS synovspace[2104]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_4" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS synovspace[2104]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS synovspace[2104]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS synovspace[2104]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS synovspace[2104]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2203]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2205]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2202]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-syno_vg_reserved_area" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2207]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2209]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2205]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2203]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2202]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-volume_5" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2209]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2207]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2202]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md6" Error="mdadm: stopped /dev/md6 "
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2209]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2203]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2205]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-09-20T11:29:09+08:00 Synology_NAS spacetool.shared[2207]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-09-20T11:33:06+08:00 Synology_NAS syno_swap_ctl[7354]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata6p2 /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 5 drives (out of 12). "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10898]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10902]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata5p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10900]: space_error_log.c:47 command="/sbin/mdadm /dev/md6 -A -u 81307ae9:3932689e:e0857486:be32e136 --run /dev/sata4p3" Error="mdadm: /dev/md6 has been started with 1 drive. "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10903]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata6p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10897]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10198]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10198]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10198]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg5" Error=""
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10198]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-09-20T11:34:38+08:00 Synology_NAS spacetool.shared[10198]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-09-20T11:34:40+08:00 Synology_NAS synovspace[11825]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 15614345216 -p dummy cachedev_0 none /dev/vg5/volume_5" Error=""
2024-09-20T11:34:40+08:00 Synology_NAS synovspace[11825]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_1 none /dev/vg4/volume_4" Error=""
2024-09-20T11:34:40+08:00 Synology_NAS synovspace[11825]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_2 none /dev/vg3/volume_3" Error=""
2024-09-20T11:34:40+08:00 Synology_NAS synovspace[11825]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_3 none /dev/vg2/volume_2" Error=""
2024-09-20T11:34:40+08:00 Synology_NAS synovspace[11825]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_4 none /dev/vg1/volume_1" Error=""
2024-09-20T11:34:41+08:00 Synology_NAS spacetool[12238]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume5" Error=""
2024-09-20T11:34:42+08:00 Synology_NAS spacetool[12272]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume3" Error=""
2024-09-20T11:34:42+08:00 Synology_NAS spacetool[12257]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume4" Error=""
2024-09-20T11:34:42+08:00 Synology_NAS spacetool[12364]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume2" Error=""
2024-09-20T11:34:42+08:00 Synology_NAS spacetool[12304]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_4 /volume1" Error=""
2024-09-20T11:34:50+08:00 Synology_NAS hotplugd[14898]: space_error_log.c:47 command="/bin/mount -t vfat -o utf8,umask=000,shortname=mixed,uid=1024,gid=100,quiet /dev/usb1p1 /volumeUSB1/usbshare1-1" Error=""
2024-09-20T11:34:57+08:00 Synology_NAS hotplugd[14898]: space_error_log.c:47 command="/bin/mount -t ntfs -o uid=1024,gid=100,big_writes /dev/usb1p2 /volumeUSB1/usbshare1-2" Error=""
2024-09-21T04:52:32+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[2714]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume4" Error=""
2024-09-21T04:52:48+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[2892]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume5" Error=""
2024-09-23T23:32:55+08:00 Synology_NAS synoscgi_SYNO.Core.Package.Installation_1_install[4086]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/web"
2024-09-23T23:32:56+08:00 Synology_NAS synoscgi_SYNO.Core.Package.Installation_1_install[4086]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/web_packages"
2024-09-26T10:47:56+08:00 Synology_NAS cloud-control[21438]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/@synologydrive"
2024-09-28T12:16:46+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[25947]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume3/Share"
2024-10-05T02:38:48+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_delete[30481]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume3/Share"
2024-10-05T21:48:33+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[11474]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume5/Season Anime"
2024-10-05T22:00:10+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[15347]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume5/NSFW"
2024-10-05T22:11:16+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[19108]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/@sharesnap"
2024-10-05T22:11:16+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[19108]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/@sharesnap/WDC WUH721414ALE604"
2024-10-05T22:12:05+08:00 Synology_NAS synoscgi_SYNO.Storage.CGI.Volume_1_set_setting[20658]: space_error_log.c:47 command="/bin/mount -o remount,relatime,relatime_period=1 /volume4" Error=""
2024-10-05T22:30:45+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[25955]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume5/@sharesnap"
2024-10-05T22:30:45+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[25955]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume5/@sharesnap/Season Anime"
2024-10-05T22:31:34+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[27150]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/Season Anime"
2024-10-05T23:22:14+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_create[11538]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/Anime"
2024-10-05T23:26:04+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_delete[17246]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume1/MG08ACA14TE"
2024-10-06T06:04:38+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[29735]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/@sharesnap"
2024-10-06T06:04:38+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[29735]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/@sharesnap/Anime"
2024-10-06T06:05:23+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_set[31072]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/@sharesnap/Season Anime"
2024-10-06T06:13:23+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_delete[5114]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume2/MG04ACA400N"
2024-10-06T06:13:50+08:00 Synology_NAS synoscgi_SYNO.Core.Share_1_delete[5669]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume5/MG08ACA16TE"
2024-10-06T06:17:11+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_create[8573]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/Comic"
2024-10-06T06:18:49+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_create[9517]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/Music"
2024-10-06T06:19:28+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_delete[10200]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume4/Comic"
2024-10-06T06:19:46+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_create[10678]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume1/Comic"
2024-10-06T06:21:33+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_create[11737]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/Movies and TV"
2024-10-06T06:26:19+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_delete[15204]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume delete /volume1/Comic"
2024-10-06T06:26:41+08:00 Yasuraokaa_Site synoscgi_SYNO.Core.Share_1_create[15779]: fs_btrfs_cmd.c:222 command="/sbin/btrfs subvolume create /volume4/Kindle"
2024-10-09T04:29:18+08:00 Yasuraokaa_Site spacetool[4238]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-10-09T04:29:18+08:00 Yasuraokaa_Site spacetool[4241]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-10-09T04:29:18+08:00 Yasuraokaa_Site spacetool[4243]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume5" Error=""
2024-10-09T04:29:18+08:00 Yasuraokaa_Site spacetool[4237]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-10-09T04:29:19+08:00 Yasuraokaa_Site spacetool[4239]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site synovspace[4342]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site synovspace[4342]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site synovspace[4342]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site synovspace[4342]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site synovspace[4342]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_4" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4455]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4453]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-syno_vg_reserved_area" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4457]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4459]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4462]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4453]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-volume_5" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4455]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4457]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4462]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4459]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4462]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4453]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md6" Error="mdadm: stopped /dev/md6 "
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4459]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4455]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-10-09T04:29:20+08:00 Yasuraokaa_Site spacetool.shared[4457]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-10-09T04:31:48+08:00 Yasuraokaa_Site syno_swap_ctl[7296]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata6p2 /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 5 drives (out of 12). "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11258]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11255]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11263]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata5p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11261]: space_error_log.c:47 command="/sbin/mdadm /dev/md6 -A -u 81307ae9:3932689e:e0857486:be32e136 --run /dev/sata4p3" Error="mdadm: /dev/md6 has been started with 1 drive. "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11265]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata6p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11032]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11032]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11032]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg5" Error=""
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11032]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-10-09T04:33:21+08:00 Yasuraokaa_Site spacetool.shared[11032]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site synovspace[11973]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 15614345216 -p dummy cachedev_0 none /dev/vg5/volume_5" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site synovspace[11973]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_1 none /dev/vg4/volume_4" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site synovspace[11973]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_2 none /dev/vg3/volume_3" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site synovspace[11973]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_3 none /dev/vg2/volume_2" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site synovspace[11973]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_4 none /dev/vg1/volume_1" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site spacetool[12438]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume2" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site spacetool[12476]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume3" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site spacetool[12412]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume4" Error=""
2024-10-09T04:33:23+08:00 Yasuraokaa_Site spacetool[12397]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume5" Error=""
2024-10-09T04:33:25+08:00 Yasuraokaa_Site spacetool[12465]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_4 /volume1" Error=""
2024-11-05T00:53:52+08:00 Yasuraokaa_Site spacetool[26152]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume4" Error=""
2024-11-05T00:53:52+08:00 Yasuraokaa_Site spacetool[26149]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume2" Error=""
2024-11-05T00:53:52+08:00 Yasuraokaa_Site spacetool[26148]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume1" Error=""
2024-11-05T00:53:53+08:00 Yasuraokaa_Site spacetool[26154]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume5" Error=""
2024-11-05T00:53:53+08:00 Yasuraokaa_Site spacetool[26150]: space_error_log.c:47 command="/bin/umount -R -A -k -f -T 600 /volume3" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site synovspace[26238]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_0" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site synovspace[26238]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_1" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site synovspace[26238]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_2" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site synovspace[26238]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_3" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site synovspace[26238]: flashcache_unload.c:158 command="/sbin/dmsetup remove --retry cachedev_4" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26339]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-syno_vg_reserved_area" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26341]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-syno_vg_reserved_area" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26338]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-syno_vg_reserved_area" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26345]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-syno_vg_reserved_area" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26343]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-syno_vg_reserved_area" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26339]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg4-volume_4" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26338]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg5-volume_5" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26341]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg3-volume_3" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26345]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg1-volume_1" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26343]: space_error_log.c:47 command="/sbin/dmsetup remove --retry /dev/mapper/vg2-volume_2" Error=""
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26338]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md6" Error="mdadm: stopped /dev/md6 "
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26345]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md2" Error="mdadm: stopped /dev/md2 "
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26339]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md5" Error="mdadm: stopped /dev/md5 "
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26341]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md4" Error="mdadm: stopped /dev/md4 "
2024-11-05T00:53:54+08:00 Yasuraokaa_Site spacetool.shared[26343]: space_error_log.c:47 command="/sbin/mdadm -S /dev/md3" Error="mdadm: stopped /dev/md3 "
2024-11-05T07:05:06+08:00 Yasuraokaa_Site syno_swap_ctl[7287]: space_error_log.c:47 command="/sbin/mdadm /dev/md1 -A -u 11a4571e:e7ac8558:05d949f7:b0bbaec7 --run /dev/sata6p2 /dev/sata5p2 /dev/sata4p2 /dev/sata3p2 /dev/sata2p2" Error="mdadm: /dev/md1 has been started with 5 drives (out of 12). "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11426]: space_error_log.c:47 command="/sbin/mdadm /dev/md5 -A -u 766a34dd:45a511b4:31ac5959:bc7fc9f6 --run /dev/sata3p3" Error="mdadm: /dev/md5 has been started with 1 drive. "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11430]: space_error_log.c:47 command="/sbin/mdadm /dev/md3 -A -u b9c78dc9:4646d410:bf4cb9f6:36450124 --run /dev/sata5p3" Error="mdadm: /dev/md3 has been started with 1 drive. "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11428]: space_error_log.c:47 command="/sbin/mdadm /dev/md6 -A -u 81307ae9:3932689e:e0857486:be32e136 --run /dev/sata4p3" Error="mdadm: /dev/md6 has been started with 1 drive. "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11431]: space_error_log.c:47 command="/sbin/mdadm /dev/md2 -A -u b840382a:4b475e33:40d635a5:4501fcf6 --run /dev/sata6p3" Error="mdadm: /dev/md2 has been started with 1 drive. "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11425]: space_error_log.c:47 command="/sbin/mdadm /dev/md4 -A -u 3f55f448:13eb18a7:86d13bfa:979108fe --run /dev/sata2p3" Error="mdadm: /dev/md4 has been started with 1 drive. "
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11174]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg3" Error=""
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11174]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg4" Error=""
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11174]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg5" Error=""
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11174]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg2" Error=""
2024-11-05T07:06:38+08:00 Yasuraokaa_Site spacetool.shared[11174]: space_error_log.c:47 command="/sbin/vgchange -ay /dev/vg1" Error=""
2024-11-05T07:06:40+08:00 Yasuraokaa_Site synovspace[11988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 15614345216 -p dummy cachedev_0 none /dev/vg5/volume_5" Error=""
2024-11-05T07:06:40+08:00 Yasuraokaa_Site synovspace[11988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_1 none /dev/vg4/volume_4" Error=""
2024-11-05T07:06:40+08:00 Yasuraokaa_Site synovspace[11988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 965738496 -p dummy cachedev_2 none /dev/vg3/volume_3" Error=""
2024-11-05T07:06:40+08:00 Yasuraokaa_Site synovspace[11988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 3895459840 -p dummy cachedev_3 none /dev/vg2/volume_2" Error=""
2024-11-05T07:06:40+08:00 Yasuraokaa_Site synovspace[11988]: flashcache_create.c:122 command="/usr/bin/flashcache_create -n 12 -s 13660848128 -p dummy cachedev_4 none /dev/vg1/volume_1" Error=""
2024-11-05T07:06:31+08:00 Yasuraokaa_Site spacetool[12450]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_2 /volume3" Error=""
2024-11-05T07:06:31+08:00 Yasuraokaa_Site spacetool[12612]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_3 /volume2" Error=""
2024-11-05T07:06:31+08:00 Yasuraokaa_Site spacetool[12437]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_1 /volume4" Error=""
2024-11-05T07:06:32+08:00 Yasuraokaa_Site spacetool[12418]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_0 /volume5" Error=""
2024-11-05T07:06:33+08:00 Yasuraokaa_Site spacetool[12546]: space_error_log.c:47 command="/bin/mount -t btrfs -o auto_reclaim_space,ssd,synoacl,relatime,nodev /dev/mapper/cachedev_4 /volume1" Error=""
